back/db/migrate/20230718214736_add_two_initiative_statuses_and_update_colors.rb

```diff
@@ -0,0 +1,68 @@
+# frozen_string_literal: true
+
+class AddTwoInitiativeStatusesAndUpdateColors < ActiveRecord::Migration[7.0]
+  class StubInitiativeStatus < ActiveRecord::Base # rubocop:disable Rails/ApplicationRecord
+    self.table_name = 'initiative_statuses'
+  end
+
+  def change
+    reversible do |dir|
+      dir.up do
+        service = MultilocService.new
+
+        proposed = StubInitiativeStatus.find_by(code: 'proposed')
+        return unless proposed
+
+        proposed.ordering = 150
+        proposed.color = '#BEE7EB'
+        proposed.save!
+
+        threshold_reached = StubInitiativeStatus.find_by(code: 'threshold_reached')
+        return unless threshold_reached
+
+        threshold_reached.color = '#40B8C5'
+        threshold_reached.save!
+
+        expired = StubInitiativeStatus.find_by(code: 'expired')
+        return unless expired
```

Comment:
Can it be that on some platforms one status is removed, but the rest should still be updated? In this case, we should not use `return`. `&` (in my example above) would still work in this case.